core/embed/extmod/modtrezorui/tjpgd/tjpgdcnf.h

```diff
@@ -0,0 +1,33 @@
+/*----------------------------------------------*/
+/* TJpgDec System Configurations R0.03          */
+/*----------------------------------------------*/
+
+#define	JD_SZBUF		512
+/* Specifies size of stream input buffer */
+
+#define JD_FORMAT		1
+/* Specifies output pixel format.
+/  0: RGB888 (24-bit/pix)
+/  1: RGB565 (16-bit/pix)
+/  2: Grayscale (8-bit/pix)
+*/
+
+#define	JD_USE_SCALE	1
+/* Switches output descaling feature.
+/  0: Disable
+/  1: Enable
+*/
+
+#define JD_TBLCLIP		0
```

Comment:
We should benchmark how faster/slower is using `JD_TBLCLIP == 1`

Reply:
optimizaton -0s
TBLCLIP 1, FASTDECODE 1 - 95.3ms
TBLCLIP 0, FASTDECODE 1 - 85.0ms
TBLCLIP 0, FASTDECODE 0 - 143.8.0ms
TBLCLIP 0, FASTDECODE 2 - 81.0ms

optimizaton -03
TBLCLIP 1, FASTDECODE 1 - 87.3
TBLCLIP 0, FASTDECODE 1 - 77.3ms
TBLCLIP 0, FASTDECODE 0 - 86.2ms
TBLCLIP 0, FASTDECODE 2 - 71.5ms

Flash space usage is up about 1.5kB when optimizing on speed (with TBLCLIP=0, FASTDECODE=1).

Fastdecode 2 requires cca 6kB more RAM.

So: TBLCLIP = 1 is useless. 

Apart from this i am leaning towards setting this -03 and FASTDECODE=1, but we probably can live with anything but FASTDECODE=0 + -Os.

Reply:
otoh maybe we could use the FASTDECOMP=2. I started using CCMRAM because of the other stuff anyway so i can place the work buffer there and enjoy a bit better performance. We can reclaim the ram anytime if we have better use for it.